Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -236,77 +235,76 @@ class GstMappedFrame {
         if (!gst_video_info_from_caps(&info, gst_sample_get_caps(sample.get())))
             return;
 
-        m_isValid = gst_video_frame_map(&m_frame, &info, gst_sample_get_buffer(sample.get()), flags);
+        gst_video_frame_map(&m_frame, &info, gst_sample_get_buffer(sample.get()), flags);
+    }
+
+    ~GstMappedFrame()
+    {
+        // TODO: Make this un-conditional when the minimum GStreamer dependency version is >= 1.22.
```

Comment:
IIRC the coding style does not allow TODOs, just FIXMEs. I guess there's philosophy about it but...